core/ui/prime/components/src/lib/autocomplete/autocomplete.component.ts

```diff
@@ -3,43 +3,123 @@ import { NG_VALUE_ACCESSOR } from "@angular/forms";
 import { InputBaseComponent } from "../input-base/input-base.component";
```

Comment:
@gitout0fhere Doublecheck: do we want the tag functionality in the existing dropdown or do we want a new component?

Reply:
Good point. It feels like a specialization of `autocomplete` and thus a separate component in itself. But I have to see the functionality. @dageorgero could you create a Storybook story for this or show me the tag component?

Reply:
I've already have the same ideea, already split the code into a new component, but it's in branch dageorge/masterdata-filter-tags! Can we let it like this in this PR's?

Reply:
https://github.com/lensbysits/core-frontend/commit/70683515f88206224dce8838ef5710f18b15d8c7
https://github.com/lensbysits/core-frontend/commit/cf123cba51ff3f2fe647f65f4395a2e6ee91a732

Reply:
I can live with that for now. Next time you better make the changes in the original branch (this branch) and merge the changes into your new branch. This way you'll never have 'outdated' code in your develop branch